malboxes.py

```diff
@@ -269,10 +275,32 @@ def spin(parser, args):
     config['profile'] = args.profile
     config['name'] = args.name
 
-    f = open("Vagrantfile", 'w')
-    f.write(template.render(config))
+    with open("Vagrantfile", 'w') as f:
+        f.write(template.render(config))
     print("Vagrantfile generated. You can move it in your analysis directory "
-            "and issue a `vagrant up` to get started with your VM.")
+          "and issue a `vagrant up` to get started with your VM.")
+
+
+def append_to_script(filename, line):
+    """ Appends a line to a file."""
+    with open(filename, 'a') as f:
+        f.write(line)
+
+
+def add_to_user_scripts(profile):
```

Comment:
As that function was intended to reduce code duplication, why wouldn't that considered refactoring ? It did not change the external behaviour so its nonfunctional changes ? 

As for the configuration files, as we talked about, the current proposition is to create a middle script called ```scripts/windows/user_scripts.ps1``` that executes the ```scripts/user/``` scripts that we could include in the ```.gitignore```. Is this what you are talking about ?